code/__DEFINES/dcs/signals/signals_hud.dm

```diff
@@ -1,2 +1,5 @@
+/// Sent from /datum/hud/proc/on_eye_change(): (atom/old_eye, atom/new_eye)
+#define COSMIG_HUD_EYE_CHANGED "hud_eye_changed"
```

Comment:
```suggestion
#define COMSIG_HUD_EYE_CHANGED "hud_eye_changed"
```

will probably also want to find and replace this everywhere

Reply:
I uh, oh I fucked up the S and M, thanks